test/src/test/resources/proto/test_schema.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto2";
+
+package org.corfudb.test;
+option java_package = "org.corfudb.test";
+
+import "corfu_options.proto";
+
+message Uuid {
+    optional uint64 msb = 1;
```

Comment:
Can I know why the both msb and lsb are optional? Does it means that if one field is missing, the value of that filed is 0?

Reply:
"optional" is the default specifier in proto3 
In this case it should really be "required" but this test is highlighting the possibility of defining protobufs in this manner.